content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -387,7 +389,7 @@ for more details.
 
 ## Known Limitations
 
-- Scaling down a Deployment may result in imbalanced Pods distribution.
+- There's no guarantee that the constraints remain satisfied when Pods are removed. For example, scaling down a Deployment may result in imbalanced Pods distribution.
```

Comment:
Totally optional: also mention descheduling and / or https://github.com/kubernetes-sigs/descheduler in the What's Next section.